Gruntfile.js

```diff
@@ -414,7 +414,8 @@ module.exports = function (grunt) {
             '*.html',
             'images/{,*/}*.{webp}',
             'styles/fonts/{,*/}*.*',
-            'data/*.json'
+            'data/*.json',
```

Comment:
Why don't you just use `data/*` or something to just copy everything within `data`?

Reply:
Done, changed so everything in data and its subfolders will be copied